requirements-dev.txt

```diff
@@ -6,4 +6,4 @@ flake8>=5.0.4
 pytest-cov>=4.0.0
 pytest-asyncio>=0.15.1
 aiounittest==1.4.2
-
+grpc-interceptor==0.15.4
```

Comment:
Does the interceptor requirement need to be added to requirements.txt for it to be accessible for library users who may not have dev requirements added?

Reply:
The interceptor requirement is only for testing in this library. The interceptor suppose is built into the core grpc library, but the interceptor package has some extra data structures that help build interceptors. But it's not a requirement.